zephyr/ra/ra_cfg/fsp_cfg/r_agt_cfg.h

```diff
@@ -0,0 +1,20 @@
+/*
+* Copyright (c) 2020 - 2024 Renesas Electronics Corporation and/or its affiliates
+*
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#ifndef R_AGT_CFG_H_
+#define R_AGT_CFG_H_
+#ifdef __cplusplus
+        extern "C" {
```

Comment:
Strange indentation.
You might want to run it through clang-format in your zephyr config.